cmd/bootstrap/transit/cmd/prepare.go

```diff
@@ -0,0 +1,50 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// prepareCmd represents a command to generate transit keys for push command
+var prepareCmd = &cobra.Command{
+	Use:   "prepare",
+	Short: "Generate transit keys for push command",
+	Long:  `Generate transit keys for push command`,
+	Run:   prepare,
+}
+
+func init() {
+	rootCmd.AddCommand(prepareCmd)
+	addPrepareCmdFlags()
+}
+
+func addPrepareCmdFlags() {
+	pullCmd.Flags().StringVar(&flagNodeRole, "role", "", `node role (can be "collection", "consensus", "execution", "verification" or "access")`)
+	_ = pullCmd.MarkFlagRequired("role")
+}
+
+// prepare generates transit keys for push command
+func prepare(cmd *cobra.Command, args []string) {
+	log.Info().Msg("running prepare")
+
+	nodeID, err := readNodeID()
```

Comment:
I would first check the role, if role != `consensus` then return else get the `nodeID` and generate the keys